src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+internal class BackupConfiguration(
```

Comment:
If it's `internal`, we should move it outside `api` subpackage, to `com.atlassian.performance.tools.jiraactions.page`

Reply:
This class will be part of the API.